staging/knative/Chart.yaml

```diff
@@ -1,20 +1,16 @@
 apiVersion: v2
 name: knative
-version: 1.10.4
+version: 1.10.5
 description: "Kubernetes-based platform to build, deploy, and manage modern serverless workloads"
 home: https://knative.dev/
 maintainers:
-- name: akirillov
 - name: adityajariwala
 icon: https://raw.githubusercontent.com/knative/docs/master/docs/images/logo/rgb/knative-logo-rgb.jpg
-appVersion: "v1.10.0"
+appVersion: "v1.10.2"
 dependencies:
   - name: eventing
-    version: 1.10.0
+    version: 1.10.1
     condition: eventing.enabled
-  - name: eventing-sources
```

Comment:
just want to wait for confirmation that removing this won't affect any customers (do you know?)

Reply:
![image](https://github.com/mesosphere/charts/assets/22758527/6fe5ffa8-896f-474f-b1cf-601c586b66b0)

Reply:
https://github.com/knative/eventing-contrib

Reply:
@gracedo only part that I removed reached EOL June 2021, all code has been moved into Knative Eventing ([GitHub post here](https://github.com/knative/community/issues/597)). I think this should be good to go

Reply:
oh, nice! and we're pulling that in from the eventing chart here, right?

Reply:
Yup!